src/common/chat/autocompleter.ts

```diff
@@ -22,9 +24,20 @@ export class Autocompleter {
       const char = entry.charAt(i);
       const maybeChild = currentNode.getChild(char);
       currentNode =
-          maybeChild !== undefined ?
```

Comment:
just curious, was this eslint?

Reply:
It was the vscode autoformatter. Apparently not required by eslint though

Reply:
Cool, I think as long as ESLint is happy either should be acceptable. I was just curious.